main.cpp

```diff
@@ -108,7 +108,52 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Person
+{
+    int age;
+    bool leftHanded;
+    float distanceTravled;
+    float leftlegLength;
+    float leftfootLength;
+    float rightlegLength;
+    float rightfootLength;
```

Comment:
These variables need initial values.
The first time you pass them somewhere, you're passing garbage values. 
You can test this by printing them out via a `std::cout` statement
- [ ] revise

Reply:
What about the other variables such as age? I've also given distanceTraveled the initial value of 0.0f as I'm guessing that needs to be initialised for the run() function to work.